firmware/usb/usb_config.h

```diff
@@ -78,17 +78,17 @@
 /* Optional callbacks from usb.c. Leave them commented if you don't want to
    use them. For the prototypes and documentation for each one, see usb.h. */
 
-#define SET_CONFIGURATION_CALLBACK app_set_configuration_callback
```

Comment:
Re-trying comment...these are a bit out of scope for the PR. Also commenting them out makes them unusable, but doesn't make a firm decision about what we want to do with them. Lets either start using them, or remove them completely

Reply:
Sounds good. Removed. Once I'm more familiar with m-stack, I'll revisit this to see if there's a performance advantage to actually using those.